src/App.js

```diff
@@ -4,13 +4,71 @@ import Footer from './Footer.js';
 import Main from './Main.js';
 import './App.css';
 import data from './data.json';
+import { ListGroup } from 'react-bootstrap';
+
+let numsArr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
 
 class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      word: '',
```

Comment:
- Consider if we need name and word? We can simplify by just focusing on horns 👍